deploy/tests/src/test/java/se/nbis/lega/cucumber/Utils.java

```diff
@@ -121,8 +121,7 @@ public String executeWithinContainer(Container container, String... command) thr
     public String executeDBQuery(String query) throws IOException, InterruptedException {
         Container dbContainer = findContainer(getProperty("container.label.db"));
 
-        String connectionString = String.format("postgresql://%s:%s@localhost/lega",
-                readTraceProperty("DB_LEGA_IN_USER"),
+        String connectionString = String.format("postgresql://lega_in:%s@localhost/lega",
```

Comment:
why change the configurable username to hard-coded `lega_in`?

Reply:
Because it is not a _configurable_ username. The username _is_ [hard-coded in the database](https://github.com/EGA-archive/LocalEGA-db/blob/master/initdb.d/grants.sql#L12).